models/order/src/order-from-quote-draft/generator.ts

```diff
@@ -0,0 +1,26 @@
+import { Reference } from '@commercetools-test-data/commons';
+import {
+  fake,
+  Generator,
+  oneOf,
+  sequence,
+} from '@commercetools-test-data/core';
+import { orderState, paymentState, shipmentState } from '../constants';
+import { TOrderFromQuoteDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/orders#orderfromquotedraft
+
+const generator = Generator<TOrderFromQuoteDraft>({
+  fields: {
+    version: sequence(),
+    quote: fake(() => Reference.random().typeId('quote')),
```

Comment:
Should we merge @jaikamat PR first and make this a ReferenceDraft which he introduced over there?

Reply:
Sounds good!

Reply:
@tdeekens It is done ✅ 🎸